server/models/Profile.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+    firstName: {
+        type:String,
+        required:true,
+        maxlength:32,
+        trim:true,   
+    }, 
+    lastName: {
+        type:String,
+        required:true,
+        maxlength:32,
+        trim:true,
+    },
+    description: {
+        type:String,
+    },
+    availability:{
+        type:[String],
+    },
```

Comment:
I don't think this is a very intuitive approach, we should ideally be working with Dates here. To put it in perspective, we'll be designing availability similar to Calendly which allows for you to setup your start time and end time Monday to Sunday.